AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -83,4 +86,14 @@ public class AlpsManager: MatchMonitorDelegate, ContextManagerDelegate {
     func contextManager(manager: ContextManager, didDetectBeacons: [CLBeacon]) {
         
     }
+    
+    // MARK: - Remote Notification Manager Delegate
+    
+    public func remoteNotificationManager(registerDeviceToken: Data) {
```

Comment:
maybe those methods could be moved to separate category?

Also I'm not entirely sure if we need facade here. `remoteNotificationManager` could be public.

Reply:
Will make RemoteNotificationManager public